docs/cep/store/index.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 1
+title: Create Stores
+---
+
+A _store_ is a place to keep your data. Stores allow creating, reading, updating, and deleting events or records stored on external data stores. This is useful for use cases when there is a need to access a common database used by various other systems, to retrieve and transfer data.
+
+## Store Syntax
+
+Stores can be created as part of your stream worker using one of the following syntaxes.
+
+### General Store Syntax
+
+You can create any store with the following syntax.
+
+```sql
+CREATE STORE <table_name> WITH(type="<store_type>", propKey=”propVal”, … , PrimaryKey='<attribute_name>', Index='<attribute_name>')(<attribute_name> <attribute_type>, ...);
+```
+
+For example:
+
+```sql
+CREATE STORE SensorTable WITH(type=’database’, collection=’SampleTable’, map.type=’json’) (sensorId string, temperature double);
+```
+
+### Table Store Syntax
+
+You can also create a table, which is a Macrometa [collection](../../collections/index.md) that is automatically a store for the stream worker that defines it.
+
+```sql
+CREATE TABLE (GLOBAL|SPOT)? <table_name> (<attribute_name> <attribute_type>, ...);
+```
+
+For example:
+
+```sql
+CREATE TABLE SensorTable (sensorId string, temperature double);
+```
+
+## Supported Stores
+
+Supported stores include:
+
+- [Macrometa collections](../../collections/index.md) (also called tables)
+- [MongoDB](mongodb.md)
+- [Redis](redis.md)
+- [RDBMS](rdbms.md)
```

Comment:
@diana-macrometa - RDBMS is too common. We do support `MySQL`, `PostgresQL`, `Oracle`.

Reply:
What do you mean by "too common"? Are you saying that MySQL, PostgresQL, and Oracle are types of RDBMS? Do we need to make separate documentation for each of these?

Reply:
@diana-macrometa - There are many RDBMS databases. We support only those from above and `SQLServer`.

Reply:
Updated. Please take a look.